README.md

```diff
@@ -1,32 +1,41 @@
-# lexical README
+# Lexical
 
-## Using the extension
-
-The extension is not yet published to the Visual Studio Marketplace. The extension must be built and installed manually.
+Lexical is a next-generation language server for the Elixir programming language.
 
-See [CONTRIBUTING.md](./CONTRIBUTING.md) for instructions on setting up dependencies.
+## Features
 
-The extension automatically downloads and installs a predefined build of Lexical from a private S3 repository. Follow [the instructions below](#using-a-version-of-lexical-other-than-the-predefined-one) if you want to make sure you're using the latest version. Support for automatic download of the latest release will be coming soon.
+- Context aware code completion
+- As-you-type compilation
+- Advanced error highlighting
+- Code actions
+- Code Formatting
+- Completely isolated build environment
 
-### Starting the extension for development
+For all the details on what makes Lexical stand out from other Elixir language servers, see the [language server repository](https://github.com/lexical-lsp/lexical).
 
-Press F5. VSCode should build and start the extension.
+## Using the extension
```

Comment:
Maybe adding a debug section after this would be better.

Like telling the user how to check the two log files: lexical.log and project.log, and why lexical sometimes crash.

Reply:
You mean you'd move the troubleshooting section before configuration? I wouldn't mind that.

Regarding why Lexical sometimes crashes, I honestly haven't been hands-on enough with Lexical itself to write something like that.

Besides, I think I'd rather keep this kind of doc in lexical itself, since it should apply to all clients, not just vscode. We could obviously link to it from here.

Reply:
> You mean you'd move the troubleshooting section before configuration? I wouldn't mind that.

Sorry, I did not notice the troubleshooting section before, ignore that suggestion.

> Regarding why Lexical sometimes crashes, I honestly haven't been hands-on enough with Lexical itself to write something like that.

Most of the crashes people are encountering now are due to erl version not being 25.2, because we did not include ERTs. A temporary solution is to modify `include_erts: false` to `include_erts: true` in `mix.exs` and override the execution path downloaded by vscode, which can solve this problem. I hope this point can be included in the troubleshooting section before we solve it.

Reply:
I added the "Known issues" section under "Troubleshooting" with info about this. Let me know if you think it needs more clarification!

Reply:
nope, thanks.